malaria24/ona/models.py

```diff
@@ -757,6 +757,18 @@ def __unicode__(self):
         return u'%s (%s)' % (self.name, self.role)
 
 
+class InboundSMS(models.Model):
+    """
+    An SMS sent to the system.
+    """
+    message_id = models.UUIDField(primary_key=True)
+    sender = models.CharField(max_length=255)
+    content = models.TextField(blank=True)
```

Comment:
I'm not entirely sure about the drf internals, but will json requests like `"content": ""` and `"content": null` both be made into a blank string for this field?

Reply:
The `null` won't. I'll fix it and add tests quickly